contracts/gov/proposals/DistributionProposal.sol

```diff
@@ -63,24 +65,25 @@ contract DistributionProposal is IDistributionProposal, Initializable {
             require(!dpInfo.claimed[voter], "DP: already claimed");
 
             uint256 reward = getPotentialReward(proposalIds[i], voter);
-            uint256 balance = address(rewardToken).thisBalance();
 
             dpInfo.claimed[voter] = true;
 
```

Comment:
Can we refactor the code below like that? Using [token balance lib](https://github.com/dexe-network/investment-contracts/blob/master/contracts/libs/utils/TokenBalance.sol#L16).

```
rewardToken.sendFunds(voter, reward);

emit DistributionProposalClaimed(proposalIds[i], voter, reward);
```

Let's remove the logic with `safeTransferFrom()` as it is really complex and unobvious.

Reply:
done